### ISSUE TITLE
scripts/start.py (example) fails with error

### ISSUE THREAD

**Issue by davidlmobley** [MobleyLab/SolvationToolkit#46]
```
File "start.py", line 53, in <module>
    builder.run( just_build = True )
  File "/Users/dmobley/anaconda/lib/python2.7/site-packages/solvationtoolkit-0.2.1.dev0-py2.7.egg/solvationtoolkit/solvated_mixtures.py", line 141, in run
    self.build_monomers()
  File "/Users/dmobley/anaconda/lib/python2.7/site-packages/solvationtoolkit-0.2.1.dev0-py2.7.egg/solvationtoolkit/solvated_mixtures.py", line 158, in build_monomers
    mol2tosdf.writeSDF(mol2_filename, sdf_filename, mol_name) # Temporarily called here (see comment below)
UnboundLocalError: local variable 'mol_name' referenced before assignment
```

Troubleshooting.

**Comment by davidlmobley** [comment]
Ah, OK, I see. The problem is that the new location of `writeSDF` occurs before  mol_name is defined in that function. I'll fix both at the same time, @camizanette .

**Comment by camizanette** [comment]
OK. Thanks!

**Comment by davidlmobley** [comment]
Resolved THIS problem with #47 . There is still another issue with start.py which I will open separately.
